aiida_quantumespresso/parsers/pp.py

```diff
@@ -0,0 +1,383 @@
+# -*- coding: utf-8 -*-
+""" Parser for pp.x calculations """
+from __future__ import absolute_import
+
+import traceback
+
+import numpy as np
+
+from aiida import orm
+from aiida.common import exceptions
+from aiida.parsers import Parser
```

Comment:
please also subclass this from the local base class i.e. `from .base import Parser`. This will give certain useful base functionality. I will point out below in the code what you can reuse

Reply:
Grand.